src/functionalTest/resources/application-functional.yaml

```diff
@@ -50,7 +50,7 @@ auth:
 test:
   url :
     refunds : ${TEST_URL:http://localhost:8080}
-    payments : ${TEST_URL_PAYMENT:http://payment-api-pr-972.service.core-compute-preview.internal}
+    payments : ${TEST_URL_PAYMENT:http://payment-api-pr-1089.service.core-compute-preview.internal}
```

Comment:
this needs to get updated to aat before merging to master

Reply:
True, will change it at the time of AAT deployment